### PR TITLE
MNT: rerender

### LINKED ISSUE #54: @conda-forge-admin rerender
### Additional comment:

_No response_

### PR DESCRIPTION
**Pull request by conda-forge-linter**
Hi! This is the friendly automated conda-forge-webservice.

I've rerendered the recipe as instructed in #54.

Here's a checklist to do before merging.
- [ ] Bump the build number if needed.

Fixes #54

### CODE REVIEW

**Comment by beckermr** [comment]
@conda-forge-admin rerender

**Comment by mgrover1** [comment]
Thanks @beckermr for suggesting these changes - waiting for the tests to run

**Comment by zssherman** [comment]
Thank you @beckermr for doing this PR! Been appreciative of the information and insight on all of this!

**Comment by beckermr** [comment]
@conda-forge-admin rerender

**Comment by beckermr** [comment]
I set up azure to store the build artifacts. We can download them and inspect them before we merge.

File: conda-forge.yml [hunk 1]
```diff
@@ -4,3 +4,5 @@ conda_forge_output_validation: true
 github:
   branch_name: main
   tooling_branch_name: main
+azure:
+  store_build_artifacts: true
```
Comment (beckermr) [inline conda-forge.yml]:
```suggestion
```

**Comment by beckermr** [comment]
@conda-forge-admin rerender

**Comment by beckermr** [comment]
Once this rerenders and passes it can be merged. I don't see any problems in the windows package I downloaded.

**Review by mgrover1 — APPROVED** [review]
Looks great - thanks @beckermr for all your help with this!!

**Comment by zssherman** [comment]
Hopefully I was looking in the right place, but in the windows win-64/arm_pyart-1.12.1 tar.bz2 files under Lib/site-packages, im only now seeing pyart and the dist-info directory.

Thank you @beckermr for all your help on this!

**Comment by zssherman** [comment]
So once this passes and can be merge, do we then just mark the windows builds for build 0 and build 1 as broken?

**Review by zssherman — APPROVED** [review]

**Comment by zssherman** [comment]
Thanks again @beckermr for doing this! I'm going to merge.

**Pull request merged**